takeDOMSnapshot.js

```diff
@@ -49,8 +54,8 @@ function getElementAssetUrls(
     }
     const srcset = element.getAttribute('srcset');
     const src = element.getAttribute('src');
-    const href =
-      element.tagName.toLowerCase() === 'image' && element.getAttribute('href');
+    const href = element.getAttribute('href');
```

Comment:
This will grab `<a href`. Maybe that's fine, but I want to make sure it is before proceeding. It might be safer to change the element tag name check to include link.

It is also worth noting that link elements are used for more than just adding stylesheets (e.g. favicons, preloading assets of any kind, etc). Since we are just working on stylesheets here, maybe it makes sense to scope this to `rel="stylesheet"`? Maybe it makes sense to get more than that at some point, but it might be best to reduce risk by keeping this tightly scoped for this change.

We should probably cover the desired behavior for `<a href` and non-stylesheet links in our tests if we aren't already.

Reply:
Great catch! I’ll update this.

Reply:
I updated the commit and added a link in the regular elements test. That will cover the extraction of asset urls and make sure that we don't incorrectly extract assets from `<a href="...">`.